textacy/tm/coherence.py

```diff
@@ -0,0 +1,203 @@
+# -*- coding: utf-8 -*-
+"""
+Work-in-progress, use at your own risk.
```

Comment:
I'd take this out for now and only add into the repo when you are confident in it.  Since it isn't implemented in `topic_model.py`, you don't need to add it in yet.  Save if for 0.2.1.